test/integration/dao/FashionKnex-test.js

```diff
@@ -11,15 +11,27 @@ async function prepareDatabase (daoHelper) {
   const knex = daoHelper.getKnex()
   const tableName = daoHelper.getTableName()
 
+  // createTableIfNotExists still runs .primary() call and leads to pkey
```

Comment:
We should consider adding this as a helper function in our project as an alternative to `createTableIfNotExists`.